app/controllers/agent.js

```diff
@@ -31,7 +32,8 @@ export default Ember.Controller.extend(DetailMixin, {
 
     actions: {
         goBack() {
```

Comment:
Could the `goBack` action live on `DetailMixin` instead of on multiple controllers?